UI/Services/WinCheckerService.cs

```diff
@@ -0,0 +1,53 @@
+using UI.Enums;
+
+namespace UI.Services;
+
+public class WinCheckerService : IWinCheckerService
```

Comment:
I think it would be better to bring the winning logic into the game entity. The `Entities.Game` can then simply have a method like `game.HasEnded()` which will bring back a `Tuple<bool, Player?> gameResult`.

This way, after making a move via [`game.Move(move)` ](https://github.com/jwankhalaf-cx/tictacgo/blob/6b52255f01374e7dd6d9732f23a41dae0dfae08e/UI/Services/GameEngine.cs#L63) we can simply do:

```
Tuple<bool, Player?> gameResult = game.HasEnded();

if (gameResult.Item1 && gameResult.Item2 is not null)
{
  // the game as ended and gameResult.Item2 is the player that won.
  // do the necessary logic
}
else
{
  // no one has won yet continue with the game
}
```